### PR TITLE
Patch task5

### PR DESCRIPTION
**Pull request by EduardoAquinta**
Completed all the error tests, needs to be verified as OK.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -94,4 +93,90 @@ describe("GET /api/review/:review_id", () => {
...
+        return request(app)
+        .patch("/api/reviews/5654343")
+        .send(reviewVoteUpdate)
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("page not found");
+        });
+    });
+    
 });
```
Comment (emilyb93) [inline __tests__/app.test.js:182]: Would like to see a test for if the inc_votes key is broken/incorrect or missing entirely such as {} or {votes: 10}

**Review by emilyb93 — CHANGES_REQUESTED** [review]
An extra test please

File: __tests__/app.test.js (hunk 1 above)
Comment (EduardoAquinta) [reply __tests__/app.test.js:182]: Will do now :)

**Review by emilyb93 — APPROVED** [review]
Looks good

**Pull request merged**